Block/Form.php

```diff
@@ -144,4 +153,11 @@ public function getCustomerCreditCardInfo(){
 
         return false;
     }
+
+    /**
+     * @return bool
+     */
```

Comment:
Is this method needed? Can we do $block->featureSwitch->isAdmin?

Feels like a not useful wrapper, but I can see us using it if it is a pattern

Reply:
@roopakv This method is needed. We can't use $block->featureSwitch->isAdmin